functions.photon.php

```diff
@@ -12,21 +12,6 @@
 function jetpack_photon_url( $image_url, $args = array(), $scheme = null ) {
 	$image_url = trim( $image_url );
 
-	if ( class_exists( 'Jetpack') ) {
```

Comment:
Need to add this to the wp.com side, but maybe with a better check. IIRC, the `Jetpack` class exists there but without the function we need.